react-big-calendar/react-big-calendar-tests.tsx

```diff
@@ -1,7 +1,7 @@
 import * as React from "react";
 import * as ReactDOM from "react-dom";
 import * as ReactDOMServer from "react-dom/server";
-import BigCalendar from 'react-big-calendar';
+import * as BigCalendar from "react-big-calendar";
```

Comment:
The documentation don't allow such usage: http://intljusticemission.github.io/react-big-calendar/examples/index.html#intro